audit/entry_formatter_test.go

```diff
@@ -144,13 +144,19 @@ func TestNewEntryFormatter(t *testing.T) {
 	}
 }
 
+type TestingHeadersAdjuster struct{}
```

Comment:
Actually, rather than create a no-op implementation of **HeadersAdjuster**, the **headersConfig** parameter can be set to `nil` in the tests. So, I'm going to remove this struct for now and later on when the **headersConfig** field is actually used and tests that rely on it are introduced, I'll bring it back.